src/test/java/com/flightstats/hub/webhook/WebhookStateReaperTest.java

```diff
@@ -179,11 +188,121 @@ void testDoesNothingIfLeadershipDisabled() throws Exception {
         assertErrorExists(webhookName);
         assertWebhookInProcessExists(webhookName);
         assertWebhookLeaderExists(webhookName);
+    }
 
-        // ...AND then clean up the state so the next test can run.  Eww.
-        when(webhookProperties.isWebhookLeadershipEnabled()).thenReturn(true);
-        reaper = new WebhookStateReaper(clusterCacheDao, contentKeysInFlight, webhookErrorService, webhookLeaderLocks, webhookProperties);
-        reaper.delete(webhookName);
+    @Test
+    void testStopCleansUpZookeeperNodesRelatedToLeadershipStateButNotProgressOrErrors() throws Exception {
```

Comment:
Oooh! Thats a very loooooooooooong test name! Since this test method does not need to be invoked manually in our code, its should be fine. It would be nice to break it up with some _ for easy reading though. Something like
testStop_cleansUpZookeeperNodeLeaderState_notProgressOrErrors

Reply:
sounds good! I'll do that for all the tests in this file.